web/server.js

```diff
@@ -0,0 +1,261 @@
+var express = require('express');
+var app = express();
+var multer = require('multer')
+var cors = require('cors');
+var fs=require('fs')
+app.use(cors())
+var bodyParser = require('body-parser');
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(express.json())
+
+//For Patient 1
+var storage = multer.diskStorage({
+    destination: function (req, file, cb) {
+      const path='TVB_patients/TVB1/raw/mri'
```

Comment:
Instead of having this hardcoded string, you could define another function that generates the proper path by receiving as argument the current patient.